.github/workflows/service_test_hdfs.yml

```diff
@@ -70,7 +70,6 @@ jobs:
           HADOOP_HOME: "/home/runner/hadoop-3.3.5"
           OPENDAL_TEST: hdfs
           OPENDAL_HDFS_ROOT: /tmp/opendal/
-          OPENDAL_HDFS_ATOMIC_WRITE_DIR: /tmp/opendal/atomic/
```

Comment:
We can copy the entire `hdfs-default` job to a new job called `hdfs-default-with-atomic-write-dir`